source/blog/2016-08-22-last-week-in-manageiq.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Last Week in ManageIQ: The Azure, nice errors, and gems"
+date: 2016-08-22 07:30 UTC
+author: juliancheal
+comments: true
+published: true
+---
+
+What ho! Welcome to another jolly edition of Last Week in ManageIQ. [Julian](https://juliancheal.co.uk) here, hailing from the Providers team. Let's put the kettle on and take a look over what's happened this last week in ManageIQ.
+
+## Featured
+
+We've had a spiffing week with [167 pull requests merged][PRs merged last week] from our marvellous contributors. ManageIQ always welcomes new contributors, we have a great list of
+[issues](https://github.com/manageiq/manageiq/issues) so take a gander to see where
+you can help us.
+
+## Improved
+
+### [The Azure, Azure](https://github.com/ManageIQ/manageiq/pull/10629)
+
+[Bronagh Sorota] (https://github.com/bronaghs) has improved the naming of the Azure regions. When creating an Azure provider with no instances; instead of naming the region `Azure-eastus`, we incorrectly called it `Azure-Azure-eastus`
+
+### [MiQ Roles API](https://github.com/ManageIQ/manageiq/pull/10452)
```

Comment:
MiQ is cased funny...it should be all caps, or spelled out ManageIQ (or just removed entirely)